core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java

```diff
@@ -55,7 +55,8 @@ public boolean areDataFieldsValid(Proposal proposal) {
             validateDataFields(proposal);
             return true;
         } catch (ProposalValidationException e) {
-            log.warn("proposal data fields are invalid. proposal={}, error={}", proposal, e.toString());
+            // TODO only log once dao state is complete
+            //log.warn("proposal data fields are invalid. proposal={}, error={}", proposal, e.toString());
```

Comment:
That should be still used. The fix in the subclass avoids the wrong logs

Reply:
force-pushed without this very change

Reply:
where is the fix that avoids that?

Reply:
Would have been better in a new commit so make it more transparent.